components/image-uploader/ImageUploader.tsx

```diff
@@ -53,7 +55,47 @@ const ImageUploader = ({ onCancel, onSuccess }: IProps) => {
         />
       )}
       {!imageInMemory && (
-        <div className="border-2 border-dashed rounded-lg bg-slate-100 px-[16px] py-[50px]">
+        <div
+          className={`border-2 border-dashed rounded-lg bg-slate-100 px-[16px] py-[50px] ${isDragging ? 'border-violet-800 border-3 animate-pulse' : 'border-slate-200'}`}
+          onDrop={(evt) => {
+            evt.preventDefault();
+            setIsDragging(false);
+            if (evt.dataTransfer.items) {
+              if (evt.dataTransfer.items.length > 1) {
+                toast.warning(
+                  'Only one file can be uploaded. the first image will be uploaded.',
+                );
+              }
+
+              const file = evt.dataTransfer.items[0].getAsFile();
+              if (file) {
+                const isPng =
+                  file.name.includes('.png') || file.name.includes('.PNG');
```

Comment:
what happens if suffix is something like .Png, .pNg, pnG etc.?
Perhaps you could lowercase the file name before creating those boolean variables.